onward/app/controllers/MostPopularController.scala

```diff
@@ -215,7 +214,7 @@ class MostPopularController(
         if (editionPopularContent.isEmpty) None
         Some(
           MostPopularCollectionResponse(
-            "Most popular",
+            "Most viewed",
```

Comment:
Should this change be part of this PR?

Reply:
I think it's subjective, but as this PR touched on rolling back and correcting/refactoring the earlier implementation start, then it's fine for it to go in with this